.github/workflows/ci.yml

```diff
@@ -80,38 +22,54 @@ jobs:
           - os: macos-13
```

Comment:
Using [macos-15](https://github.com/actions/runner-images/blob/main/images/macos/macos-15-Readme.md) would probably build the macOS arm64 wheels much faster. This could be in the next release cycle.